build/args/all.gn

```diff
@@ -0,0 +1,8 @@
+is_electron_build = true
+is_electron_gn_build = true
+root_extra_deps = [ "//electron" ]
+
```

Comment:
should we also add `is_mas_build = getenv("MAS_BUILD") != ""` here ?

Reply:
no. we should not depend on environment variables in the GN build, otherwise the whole point of `args.gn` would be defeated.

If you want to build a MAS build, set `is_mas_build = true` in your args.gn.